source/main.lua

```diff
@@ -116,8 +116,8 @@ function love.load()
 	garrImages[1] = newImage(path .. "tower.png")
 	garrImages[2] = newImage(path .. "gastank1.png")
 	garrImages[3] = newImage(path .. "background1.png")
-	garrImages[4] = newImage(path .. "flame.png")
-	garrImages[5] = newImage(path .. "ship.png")
+	garrImages[enum.imageFlameSprite] = newImage(path .. "flame.png")

```

Comment:
I don't like the inconsistency here, but this will be solved with the assetloader anyway so who cares :stuck_out_tongue_closed_eyes: